### ISSUE TITLE
[ISSUE] Freece after Tool Change Detection 

### ISSUE THREAD

**Issue by Knarf2807** [howie-j/OpenSolder#5]
If the MCU detects the Tool change, the station freeces. The temperature in the display does not drop. The "energy bar" freece and I can't change the target temperatur.

It takes some seconds until unfreece.

At the "handpeace in stand" - event it works fine.

![IMG20230207204457.jpg](https://user-images.githubusercontent.com/58421457/217351393-af4f43b0-53aa-4441-845d-42ab1d50dcd6.jpg)

**Comment by howie-j** [comment]
Yes that's correct. I did not think of this as an issue, as i never leave the tool/handpiece in the tip change bracket/sensor, i only use it to instantly remove the tip. And when the tip is removed the display changes from default display to display_message until a tip is inserted. The delay is there to be able to insert a new tip by hand and not burn your fingers🤕 The delay can also be modified in the opensolder.h file.

If you have a recommendation on how to improve the gui and program flow, please share! And if you want to play around yourself, the [TIP_CHANGE_STATE in opensolder.c line 85 and downwards.](https://github.com/howie-j/OpenSolder/blob/main/firmware/opensolder/Core/Src/opensolder.c#L85)

**Comment by Knarf2807** [comment]
That's a good advice. And I think you are right. In practice it's a feature, not a bug. 

Perhaps (impovement with low prio) the frozen display can be changed. Similar to the "handpiece in stand"-event. I will try. Perhaps my programming-skills are good enough.
